app/assets/javascripts/i18n/translations.js

```diff
@@ -17133,6 +17133,7 @@ I18n.translations["en"] = {
     "projects": {
       "collection": "Collection",
       "edit": {
+        "admins_must_be_existing_members": "Admins must be existing members of the project.",
```

Comment:
This needs to be specified in https://github.com/inaturalist/inaturalist/blob/master/config/locales/en.yml#L5675. This js file will get overwritten with every deploy.

Reply:
Ahh, I see `generate_translations_js` rake task. Thanks for the info. I guess changing this output is irrelevant and unnecessary.